src/cockpit/bridge.py

```diff
@@ -270,6 +271,9 @@ def main(*, beipack: bool = False) -> None:
     if args.packages:
         Packages().show()
         return
+    elif args.version:
+        print(f'Version: {__version__}\nProtocol: 1')
```

Comment:
The C bridge looks like this:
```
Version: 287
Protocol: 1
Payloads: dbus-json3 http-stream1 http-stream2 stream packet fsread1 
    fsreplace1 fswatch1 fslist1 null echo websocket-stream1 
Authorization: crypt1
```

The "Authorization: crypt1" is hardcoded in the C bridge, and thus not really useful. We could implement "Payloads:" if we care, but I doubt it's actually useful.

Reply:
For some reason the inconsistency of how we get the two newlines printed out is bothering me.  I will ignore that little nag, though. :)